docker-compose.yml

```diff
@@ -33,7 +33,6 @@ services:
         - ./deploy/config/local/server.conf:/etc/nginx/conf.d/default.conf
         - ./deploy/config/nginx.conf:/etc/nginx/nginx.conf
         - ./public/index.php:/var/www/html/public/index.php
```

Comment:
Will this still work locally from a fresh clone?

I think we don't have a script or a line in Dockerfile to get `wp-admin/index.php` into the nginx-dev container.